build/stf-run-ci/tasks/setup_base.yml

```diff
@@ -66,6 +80,28 @@
             source: redhat-operators
             sourceNamespace: openshift-marketplace
 
+# deploy cert-manager from stable-v1 in 4.12 and later using namespace scoped operator
+- when: not __deploy_from_index_enabled | bool and ocp_ver.stdout is version ('4.12', '>=')
```

Comment:
I took a bit of liberties in this file and refactored this so that in installations of OCP 4.10 (and 4.11) we use the tech-preview channel as a cluster scoped Operator, and in 4.12 and later using the stable-v1 channel in a namespace scoped Operator to simulate how dependency resolution would work in an OCP without cert-manager already installed at the cluster scope.